### DISCUSSION TITLE
Qol suggestions

### DISCUSSION THREAD

**Discussion by HierarchyMan** [diamond-rip/Eden#93] [Ideas]
- Command to queue any kit without gui menu
Eg: /queue UNRANKED bedfight solos
- Duel accept command
Eg: /duel accept {sender}
- Command to select arena without gui in /duel
Eg: /duel {someone} -a {arenaname}
- ping range for ranked queue
- /nospeed command only in boxing kit
- after death don't tp spectators to spawn keep them where they are or let us chose spectator spawn

Question
How do I enable arena selection for duels. It isn't in /settings

**Comment by ABIGDachshunch** [comment]
you can turn the ability to see player in lobby in the config.yml

**Reply by HierarchyMan** [reply]
Ty

**Comment by GoodestEnglish** [comment]
> * Command to queue any kit without gui menu
>   Eg: /queue UNRANKED bedfight solos
> * Duel accept command
>   Eg: /duel accept {sender}
> * Command to select arena without gui in /duel
>   Eg: /duel {someone} -a {arenaname}
> * ping range for ranked queue
> * /nospeed command only in boxing kit
> * after death don't tp spectators to spawn keep them where they are or let us chose spectator spawn
> 
> Question How do I enable arena selection for duels. It isn't in /settings

It is in settings, the second option (which icon is a dirt block)
